test/models/report_test.rb

```diff
@@ -48,7 +48,7 @@ class ReportTest < ActiveSupport::TestCase
     result = r.index_data
     assert_equal Department.count, result['departments'].pluck(:label).length
     assert_includes result['departments'].pluck(:label), Department.first.name_dw
-    assert_equal result['departments'][0][:data].values, [0, 3, 0, 0, 0, 0, 0, 4, 0]
+    assert_equal result['departments'][0][:data].values, [0, 2, 0, 0, 0, 0, 0, 3, 1]
```

Comment:
This values changed and I have no idea what they mean and if they should have. I'm okay with not understanding these tests for now if you have thought about why these values changed and they are appropriate (this applies to all of the changing numbers in this test file).

Reply:
I stared at this for about an hour, and I'm 75% sure that these changes are due to the changes in the fixtures.